rest-api/recipe/routes/recipe-routes.js

```diff
@@ -16,4 +19,8 @@ router.get('/', getRecipes);
 router.get('/:id', validateId, getRecipeById);
 router.post('/', validateToken, requiredFields, addRecipe);
 
+//sam
+router.put('/recipe/:id', validateId, requiredFields, editRecipe)
```

Comment:
This path is wrong. `/recipe` is handled by the router already. Should be just `/:id`! :))

Reply:
I agree as a matter of fact! So editing the first recipe object means we'll need to add a specific path. For example, `/:id/recipeinfo` ?? what do you think?

Reply:
Yeah, something like that! Perhaps `/:id/recipe-info`? :)